image-cmd.py

```diff
@@ -6,6 +6,8 @@
 #- displays that image to the terminal using shitty color terminal graphics
 
 import os
+import sys
```

Comment:
This import is unused. This is an example of "dead code". Dead code can sometimes lead to bugs. Please remove the import.